cordial_behavior/scripts/behaviors.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env python
+
+import rospy
+import sys
+import actionlib
+import json
+from ast import literal_eval
+from std_msgs.msg import String, Bool
+from cordial_behavior.msg import Behavior
+from qt_robot_gestures.msg import Gesture
+from cordial_face.msg import FaceRequest
+from cordial_manager.msg import InteractionFeedback, InteractionAction, InteractionResult
+from qt_robot_speaker.msg import PlayRequest # Rename it! PlayAudio
+from threading import Timer
+import os
+
+RATE_FEEDBACK =""
+#data = "[{'start': 0.075, 'time': 2,'type': 'action', 'id': 'QT/bye'},{'start': 0.075, 'time': 2,'type': 'action', 'id': 'QT/point_front'}, {'start': 0.075,'time': 2, 'type': 'viseme', 'id': 'POSTALVEOLAR'},{'start': 0.006, 'time': 2,  'type': 'action', 'id': 'happy_face'}]"
+TTS_MESSAGE = ''
+SPEAKER_DONE = False
+TRACKER_STATE = ''
+TRACKER_DONE = False
+DETECTOR_MESSAGE = ''
+
+
+FEEDBACK_MESSAGE = ''
+INTERACTION_MESSAGE = ''
+INTERACTION_CONTINUE = True
+
+class LongBehaviorServer():
```

Comment:
I might have to ask about long behaviors--my intuition is that it is things like tracking or logging or something, is that right?